packages/math-input/src/full-mobile-input.stories.tsx

```diff
@@ -104,10 +110,17 @@ export function Wrapped() {
     return (
         <StatefulKeypadContextProvider>
             <KeypadContext.Consumer>
-                {({keypadElement, setKeypadElement}) => (
+                {({
+                    keypadElement,
+                    setKeypadElement,
+                    keypadActive,
+                    setKeypadActive,
+                }) => (
                     <Basic
                         keypadElement={keypadElement}
                         setKeypadElement={setKeypadElement}
+                        keypadActive={keypadActive}
+                        setKeypadActive={setKeypadActive}
```

Comment:
To avoid breaking compatibility with existing callers, it might be better to wrap the MobileKeypad in a component that uses the `KeypadContext.Consumer` to get `keypadActive` and `setKeypadActive`. @handeyeco do you think that would be an improvement?

Reply:
Yeah, that's definitely the way I'd like to go. See [this ticket](https://khanacademy.atlassian.net/browse/LC-1340):

> What’s even nicer is that this could likely all happen within Perseus - rather than passing things around in Webapp, we can pass things around in Perseus using Context; the Perseus widget passes a config up to the Perseus context which passes the config down to the Perseus keypad.